website/package.json

```diff
@@ -8,7 +8,7 @@
     "@hashicorp/mktg-logos": "1.0.2",
     "@hashicorp/nextjs-scripts": "18.2.0",
     "@hashicorp/react-button": "5.0.0",
-    "@hashicorp/react-docs-page": "13.1.3",
+    "@hashicorp/react-docs-page": "13.1.7-alpha.10",
```

Comment:
Will bump this to a stable release once these PRs are finalized and https://github.com/hashicorp/react-components/pull/220 is merged and published.

Reply:
✅ Bumped to stable